weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -1430,11 +1433,12 @@ def _select_objs_query(
                         ) AS rn
                     FROM object_versions
                     WHERE project_id = {{project_id: String}}
+                       AND {conditions_part}
                 )
                 WHERE rn = 1
             )
-            WHERE project_id = {{project_id: String}} AND
```

Comment:
why do we need this?

Reply:
abundance of caution - probably left over. can never let things get through that are not the right project for security reasons